packages/lit-html/rollup.config.js

```diff
@@ -30,6 +30,9 @@ export default {
       },
       output: {
         comments: false,
```

Comment:
Just an FYI that I've been using the default value for `comments` in my sample Rollup configs; the default tries to retain license comments. This does leave us with a good number of duplicate license comments, and hence a larger code size. But perhaps we should make the two match one way or the other.

Reply:
When we publish the bundle I think we'll have to include a license comment, but we'll ensure there's only one, so this shouldn't be a problem.

For 1.x we should have a script that makes sure all our comments are dedupable.